spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteTablePathSparkAction.java

```diff
@@ -239,7 +239,7 @@ private boolean versionInFilePath(String path, String version) {
   }
 
   private String jobDesc() {
-    if (startVersionName != null) {
+    if (startVersionName == null) {
```

Comment:
Nice catch! Can we add a UT?

Reply:
That's overkill to me.

Reply:
Yea I dont think we have too many tests asserting a message, Im fine without it as well.

Reply:
I'm ok with the current status, but I still believe we could have caught it with a UT.